app/components/product-show/template.hbs

```diff
@@ -1,4 +1,17 @@
 
+<div>
```

Comment:
I think you don't need div here (this is a default wrapper element for all components in Ember)